CSS/firstjump.css

```diff
@@ -0,0 +1,180 @@
+body {
+  background-color: white;
+  padding: 0px;
+  margin: 0px;
+  width: 100%;
+}
+
+/* Font section */
+
+.site_font {
+  font-family: "Roboto Condensed", sans-serif;
+}
+
+.site_font.bold {
+  font-weight: 700;
+}
+
+.site_font.light {
+  font-weight: 300;
+}
+
+.site_font.regular {
+  font-weight: 400;
+}
+
+.site_alt_font_one {
+  font-family: cursive;
+  font-size: 24px;
+}
+
+.site_alt_font_two {
+  font-size: 24px;
+}
+
+/* Banner */
+
+#bannertitle {
+  font-size: 7vw;
+  text-align: center;
+  color: white;
+  background-color: #4282d6;
+}
+
+.banner_title {
+  margin-top: 0%;
+  background-color: #4282d6;
+  padding-top: 10%;
+}
+
+#bannerImageContainer {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  position: relative;
+  background-color: #4282d6;
+}
+
+#bannerClouds {
+  width: 100%;
+  margin-top: 30%;
+  background-size: cover;
+  overflow: hidden;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  position: relative;
+}
+
+#bannerSkydiver {
+  width: 80%;
+  position: absolute;
+  margin-left: 12%;
+  margin-bottom: 5%;
+}
+
+#infoSection {
+  margin-top: -2%;
+  background-color: white;
+  width: 100%;
+  height: 800px;
+  /* display: flow-root; */
+  margin-left: 5%;
+}
+
+#parachute {
+  float: right;
+  width: 40%;
+  display: flow-root;
+  margin-right: 0%;
+}
+#section {
+  margin-top: 50%;
+  margin-left: 5%;
+}
+
+#formHeader {
+  text-align: center;
+}
+
+#form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 60%;
+  margin-left: 20%;
+  margin-top: 10%;
+  font-size: 2vw;
```

Comment:
I almost criticized your for using VW on fonts but shows what I know. This doesnt look quite right though, so lets go over it together. Heres an article for both of us to read: https://www.smashingmagazine.com/2016/05/fluid-typography/

Smashing magazine btw is a great source. This is from 2016 which is ancient in tech years so if you have a better article for VW on fonts please suggest